view-transitions/mpa/styles.css

```diff
@@ -0,0 +1,73 @@
+/* Example layout */
+
+html {
+    font-family: Arial, Helvetica, sans-serif;
+    height: 100%;
+}
+
+body {
+    margin: 0;
+    height: inherit;
+}
+
+img {
+    width: 100vw;
+    height: 100vh;
+    object-fit: cover;
+}
+
+p {
+    position: absolute;
+    top: 20px;
+    left: 20px;
+    font-size: 2rem;
+    padding: 20px;
+    margin: 0;
+    border-radius: 5px;
+    background-color: rgba(255 255 255 / 0.8);
+}
+
+/* Turn cross-document view-transitions on */
+/* Note that this at-rule is all that is needed to create the default cross-fade animation  */
+
+@view-transition {
+    navigation: auto;
+}
+
+/* Customize the default animation */
+
+/* ::view-transition-old(root),
+::view-transition-new(root) {
+    animation-duration: 0.5s;
+} */
```

Comment:
I wonder if it would be better to enable this customization by default. This would invite you to play with values and explore the behavior right in the DevTools.

Reply:
Good idea; I've done this in my next commit.